bin/opteadm/build.rs

```diff
@@ -0,0 +1,31 @@
+use std::error::Error;
+use std::process::Command;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    println!("cargo:rerun-if-changed=../../.git");
+
+    let commit_count_out =
+        Command::new("git").args(["rev-list", "--count", "HEAD"]).output()?;
```

Comment:
Is there any issue with pointing to HEAD rather than, say `main`? I think we always build CI artifacts from `main`, but just checking.

Reply:
`master`/`main` would probably be more correct here -- at least for preventing odd version numbers in development branches -- but HEAD is what is used by `pkg/build.sh`. We could change it, we'd just need to do so in both spots.